### ISSUE TITLE
"can't find neuron executable"

### ISSUE THREAD

**Issue by zhenjieliang** [oberblastmeister/neuron.nvim#41]
run the command in neovim

:lua  require 'neuron'.setup()

return the error "neuron is not executable".

But I had installed the neuron already and run command in bash "neuron -v" works.

**Comment by arkottke** [comment]
Do you have a space between require and '?

**Comment by zhenjieliang** [comment]
> Do you have a space between require and '?

Thanks for attention. Yes, I do have a space between require and '

**Comment by zhenjieliang** [comment]
How can I add the neuron binary path to the neovim executable search paths?

**Comment by lokesh-krishna** [comment]
You just need to add it to your `$PATH`.

**Comment by zhenjieliang** [comment]
> You just need to add it to your `$PATH`.

Thanks for reply! Did you mean add the neuron binary path in my ~/.zshrc and then source it?

Already done it

export PATH=/nix/store/42kyavaxgcmxisxap302fssyvyn39n66-neuron-1.9.35.3/bin:$PATH

but still the problem remains.

**Comment by lokesh-krishna** [comment]
Does `echo $PATH` show this location? As long as I had the binary in one of the locations, I could get it to work.

**Comment by zhenjieliang** [comment]
> Does `echo $PATH` show this location? As long as I had the binary in one of the locations, I could get it to work.

Yes, I run the command in terminal, and the neuron binary path does show up. But still the problem remains.

**Comment by AuLaSW** [comment]
Has anybody found a solution to this? I've been experiencing the same problem and cannot get it to work.
